.github/workflows/test_araroot.yml

```diff
@@ -0,0 +1,68 @@
+name: test_araroot
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+
+      - name: Install dependencies
+        run: |
+          
+          sudo apt-get install dpkg-dev cmake g++ gcc binutils libx11-dev \
+           libxpm-dev libxft-dev libxext-dev python libssl-dev libfftw3-dev libgsl-dev libboost-dev
+
+
+      - name: Install ROOT
+        run: |
+          
+          wget -q https://root.cern/download/root_v6.22.06.Linux-ubuntu20-x86_64-gcc9.3.tar.gz
+          tar -xzf root_v6.22.06.Linux-ubuntu20-x86_64-gcc9.3.tar.gz
+          source root/bin/thisroot.sh
+          root --version
+
+
+      - name: Install libRootFftwWrapper
+        run: |
+          
+          source root/bin/thisroot.sh
+
+          # download
+          wget -q https://github.com/nichol77/libRootFftwWrapper/archive/master.tar.gz -O libRootFftwWrapper.tar.gz
+          mkdir libRootFftwWrapper
+          tar -xzf libRootFftwWrapper.tar.gz -C libRootFftwWrapper --strip-components=1
+          rm libRootFftwWrapper.tar.gz
+          
+
+          # prepare to build
+          cd libRootFftwWrapper
+          sed -i 's:@ccmake:@cmake:' Makefile
+
+          # now build
+          make configure
```

Comment:
you can just make and it will do the default configuration :) (or at least... it *should*). This also obviates the sed line.

Reply:
You mean I should be able to remove both the sed line and the make configure line?

Reply:
yes! that should work.

Reply:
Indeed you are right. I simplified this in #958df9a. So I think that's good.